app/src/test/java/com/marvelapp/presentation/viewmodel/MainMenuViewModelTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.marvelapp.presentation.viewmodel
```

Comment:
For this PR this is not necessary as the ManiMenu should be totally empty

Reply:
Solved